spec/Driver/Driver.vspec

```diff
@@ -43,7 +43,12 @@ FatigueLevel:
   unit: percent
   min: 0
   max: 100
-  description: Fatigue level of the driver, which can be evaluated by multiple factors e.g. trip time, behaviour of steering, eye status.
+  description: Fatigue level of the driver, which can be evaluated by multiple factors. For example, 
```

Comment:
This shouldn't no be "just" a yaml comment, as those will likely never be seen. generally I think having a crisp ~1 sentence "description" and then more  verbose information in a "comment" attribute is the way to go.

Reply:
Further thoughts. Is this signal indeed reflecting the driver's fatigue level or instead the probability that the driver is fatigued? It sure seems that the probability would be more of interest than the actual level. If indeed this is the probability, the signal would need to be renamed and the description would need to refer to probability instead of level.